server/test/repository/VersionRepositoryTest.java

```diff
@@ -219,6 +219,46 @@ public void testPublishWithOnlyQuestionsModified() {
     assertThat(oldActive.getLifecycleStage()).isEqualTo(LifecycleStage.OBSOLETE);
   }
 
+  @Test
+  public void testPublishWithOnlyQuestions() {
```

Comment:
**suggestion:** Could you name this a bit more descriptively? The one before is called `testPublishWithOnlyQuestionsModified` and it's hard to tell what's different here without looking more closely.

Reply:
Done, hope that's clearer